zinc/src/main/scala/sbt/internal/inc/LookupImpl.scala

```diff
@@ -55,6 +56,13 @@ class LookupImpl(compileConfiguration: CompileConfiguration, previousSetup: Opti
     .flatMap(ext => ext.getExternalLookup().toOption)
     .collect { case externalLookup: ExternalLookup => externalLookup }
 
+  override def lookupAnalyzedClass(binaryClassName: String): Option[AnalyzedClass] = {
+    externalLookup match { // not flatMap so that external lookup can fast-track returning None
```

Comment:
I'm trying to integrate this change into sbt, and I'm a bit confused. What does it mean for `externalLookup.lookupAnalyzedClass(binaryClassName)` to return `None` here?
sbt already implements `externalLookup`, so I would like a way to say I don't want to participate AnalyzedClass lookup, but if I return `None` it would be interpreted as "I couldn't find the AnalyzedClass"?

Reply:
Yeah, the current setup has no way for `externalLookup.lookupAnalyzedClass` to say "I don't know, you try".  You'd have to modify this if you need that.

I'm not sure why `Some(APIs.emptyAnalyzedClass)` wasn't used for fast-track's "I got you, here's the API for that class name".